draft-js-dnd-plugin/src/components/block-alignment-wrapper.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Entity } from 'draft-js';
+
+const getDisplayName = (WrappedComponent) => (
+    WrappedComponent.displayName || WrappedComponent.name || 'Component'
+);
+
+// Export
+export default function WrapComponent(C) {
+  return class Wrapper extends Component {
+    static displayName = `Decorated(${getDisplayName(C)})`;
+    static defaultProps = {
+      draggable: true,
+      readOnly: false,
+    };
+    align(alignment) {
+      const entityKey = this.props.block.getEntityAt(0);
+      if (entityKey) {
+        Entity.mergeData(entityKey, { alignment });
+
+        // Force refresh
+        this.props.blockProps.refreshEditorState();
+      }
+    }
+
+    render() {
+      const { blockProps } = this.props;
+      return (
+          <C {...this.props}
+            alignment={blockProps.alignment}
+            align={::this.align}
```

Comment:
Binding inside the render function would cause a re-binding on ever render. That's why we opted in for the syntax with arrow functions:

```
align = (alignment) => {
  …
}
```